lib/active_record_view_model/update_operation.rb

```diff
@@ -360,11 +362,25 @@ def build_updates_for_collection_association(association_data, association_updat
           association_update.values
 
         when CollectionUpdate::Type::Functional
-          child_datas =
+          child_datas        =
```

Comment:
It's the IDE aligning it with the next assignment statement. I think the IDE style is wrong in this case, reduced to one space.